app/components/chart-horz-bar.js

```diff
@@ -0,0 +1,85 @@
+import Ember from 'ember';
+
+import { select } from 'd3-selection';
+import { scaleOrdinal, scaleBand, scaleLinear, schemeCategory10 } from 'd3-scale';
+import { max } from 'd3-array';
+import { axisBottom, axisLeft } from 'd3-axis';
+
+const { Component, run, get } = Ember;
+
+export default Component.extend({
+  tagName: 'svg',
+  classNames: ['chart-horz-bar'],
+  attributeBindings: ['width', 'height'],
+  didReceiveAttrs() {
+    // Anytime we get an update schedule a draw
+    run.scheduleOnce('render', this, this.draw);
+  },
+  data: null,
+  width: null,
+  height: null,
+  draw(){
+    const data = get(this, 'data');
+    const dataOrArray = data?data:[];
+    const svg = select(this.element);
+    const margin = {top: 10, right: 20, bottom: 30, left: 25};
+    const width = get(this, 'width');
+    const height = get(this, 'height');
+    const chartWidth = width - margin.left - margin.right;
+    const chartHeight = height - margin.top - margin.bottom;
+    const color = scaleOrdinal(schemeCategory10);
+
+    const x = scaleLinear().range([chartWidth, 0]);
+    const y = scaleBand().range([0, chartHeight]).padding(0.4);
+
+    if (dataOrArray.length === 0) {
```

Comment:
Will make those changes and start a new branch for turning off labels for all charts.